src/sbt-test/docker/override-commands/build.sbt

```diff
@@ -9,7 +9,7 @@ version := "0.1.0"
 maintainer := "Gary Coady <gary@lyranthe.org>"
 
 dockerCommands := Seq(
-  Cmd("FROM", "dockerfile/java:latest"),
+  Cmd("FROM", "dockerfile/openjdk:latest"),
```

Comment:
I guess at the time of this commit `dockerfile/openjdk` was still valid. 🍒-picking was the write thing 😊